### PR TITLE
Create uitests.yml

### PR DESCRIPTION
**Pull request by jwdomes**

### CODE REVIEW

File: .github/workflows/uitests.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install npm dependencies
+      run: npm install --verbose
+      working-directory: ./thea-ui
+    
+    - name: Run linter
+      run: npm run lint
```
Comment (tlazar) [inline .github/workflows/uitests.yml:21]: I believe some files are failing the linter when I run this locally.

Comment (jwdomes) [reply .github/workflows/uitests.yml:21]: Did you use the "pointless change" push to test the GitHub action or did you run it locally using a different means?

Comment (tlazar) [reply .github/workflows/uitests.yml:21]: I ran it locally with `npm run lint`.

Comment (jwdomes) [reply .github/workflows/uitests.yml:21]: Is it failing because of trailing whitespace? A few of the `.spec` files have that issue.

**Pull request merged**